workflow/usage.rst

```diff
@@ -450,3 +459,220 @@ The following example shows these functions in action:
+                    <!-- ... -->
+                </framework:workflow>
+            </framework:config>
+        </container>
+
+    .. code-block:: php
+
+        // config/packages/workflow.php
+
+        $container->loadFromExtension('framework', array(
+            // ...
+            'workflows' => array(
+                'blog_publishing' => array(
+                    'metadata' => array(
+                        'title' => 'Blog Publishing Workflow',
+                    ),
+                    // ...
+                    'places' => array(
+                        'draft' => array(
+                            'metadata' => array(
+                                'max_num_of_words' => 500,
+                            ),
+                        ),
+                        // ...
+                    ),
+                    'transitions' => array(
+                        'to_review' => array(
+                            'from' => 'draft',
+                            'to' => 'review',
+                            'metadata' => array(
+                                'priority' => 0.5,
+                            ),
+                         ),
+                     ),
+                 ),
+             ),
+         ));
+
+Then you can access this metadata in your controller as follows::
+
+    public function myControllerAction(Registry $registry, Article $article)
+    {
+        $workflow = $registry->get($article);
+
+        $workflow
+            ->getMetadataStore()
+            ->getWorkflowMetadata()['title'] ?? false
+        ;
+
+        // or
+        $workflow
+            ->getMetadataStore()
+            ->getPlaceMetadata('draft')['title'] ?? false
+        ;
+
+        // or
+        $aTransition = $workflow->getDefinition()->getTransitions()[0];
+        $workflow
+            ->getMetadataStore()
+            ->getTransitionMetadata($aTransition)['title'] ?? false
+        ;
+    }
+
+There is a shortcut that works with everything::
+
+    $workflow
+        ->getMetadataStore()
+        ->getMetadata('title')
+    ;
+
+In a Flash message in your Controller::
+
+            // $transition = ...; (an instance of Transition)
+            // $workflow is a Workflow instance retrieved from the Registry (see above)
+            $title = $workflow->getMetadataStore()->getMetadata('title', $transition);
+            $this->addFlash('info', "You have successfully applied the transition with title: '$title'");
+
+In a listener, access via the Event
+
+Metadata can also be accessed in a Listener, from the Event object.
+
+The example below uses a new feature introduced in 4.1 called Transition Blockers. These let you
+return a user-friendly error message when you stop a transition from happening. In the example we
+get this user-friendly message from the Event's metadata, giving you an easy place to manage the
+text. This is a contrived example; in production systems you may prefer to use the
```

Comment:
"unreal", "hypothetic", "fictive" ... ? may be more friendly than "contrived". Personally I'm glad to have learned that word, thanks :D.

Reply:
Contrived is the most accurate word out of the four. Hypothetical doesn't have quite the same meaning.

Reply:
I'm afraid we can't use "contrived" because it's a complex English word (lots of non native speakers don't understand it, including myself). Please, replace it by any of these synonyms: https://www.thesaurus.com/browse/contrived Thanks!

Reply:
"fake" sounds good.

Reply:
> "fake" sounds good.

It's not a fake example: it's a real example (but one people should not emulate :) ). The synonyms are not accurate for this meaning (they are poor synonyms), so this must be totally rephrased.

Elsewhere in the documentation you must point out that an example is simplified and should not be used in production. Can you point me to the wording used?

Reply:
Resolving here to carry on in the new comment below